.travis.yml

```diff
@@ -154,7 +154,7 @@ matrix:
       env:
         - SHARD="Self checks, lint, and JVM tests"
       script:
-        - ./build-support/bin/ci.sh -fkmrjt "${SHARD}"
+        - ./build-support/bin/ci.sh -fkmrjst "${SHARD}"
```

Comment:
The lint shard is not one of the shortest... while it's a bit odd, I'd wonder whether it would make more sense to do this on the rust tests shard.

Reply:
Yeah, I was unsure between a new shard, the rust tests shard, and the lint shard...

The disadvantage of the test shard is that it conflates style issues with test issues, but I guess that's fine. I'm kind of tempted to just make it its own shard; there's very little set-up to do, now that we cache the rust toolchains...